ci-operator/jobs/openshift/assisted-test-infra/openshift-assisted-test-infra-master-presubmits.yaml

```diff
@@ -1246,6 +1246,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-single-node-live-iso,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
this means it will run for each PR in assisted-test-infra, while still not being required for the PR to merge
I'm okay with that, just wanted to make sure you're aware

Reply:
This configuration is auto generated by "make jobs" command. 
For some reason "make update" didn't update this code as expected.
running "make jobs" manually did the trick. nice catch.